xbmc/listproviders/DirectoryProvider.cpp

```diff
@@ -495,7 +496,18 @@ class CVideoSelectActionProcessor : public VIDEO::GUILIB::CVideoSelectActionProc
 
   bool OnPlaySelected() override
   {
-    ExecuteAction({"PlayMedia", m_item, "noresume"});
+    if (m_item.GetVideoInfoTag()->m_type == MediaTypeMovie)
```

Comment:
Could you please add a `//! @todo get rid of special handling for movie versions`, so we can pick this up in a follow-up PR?